coc/events.py

```diff
@@ -483,23 +592,57 @@ def remove_clan_updates(self, *tags):
             except KeyError:
                 pass  # tag didn't exist to start with
 
+    def add_raid_updates(self, *tags):
+        """Add clan tags to receive updates for.
+        Parameters
+        ----------
+        \\*tags : str
+            The clan tags to add. If you wish to pass in an iterable, you must unpack it with \\*.
+        Example
+        -------
+        .. code-block:: python3
+            client.add_raid_updates("#tag1", "#tag2", "#tag3")
+            tags = ["#tag4", "#tag5", "#tag6"]
+            client.add_raid_updates(*tags)
+        """
+        for tag in tags:
+            if not isinstance(tag, str):
+                raise TypeError("clan tag must be of type str not {0!r}".format(tag))
```

Comment:
We decided to switch to f strings with everything that we touch, so changing this as well would be great.